tools/dbTools.py

```diff
@@ -115,9 +124,21 @@ def is_scraped(self, file_hash):
         )
         result = self.cursor.fetchone()
         if result:
-            return result.id, result.scrape_again
+            self.logger.info('Got a result')
+            self.logger.info(result.scrape_again)
+            self.logger.info(type(result.scrape_again))
+            self.logger.info(dir(result.scrape_again))
```

Comment:
The `dir()` and `type()` seem like odd things to log.

If this is a type defined in an external library, the `type()` and `dir()` are unlikely to change on subsequent results, surely?

And if this is a custom type, define your own `__repr__` method and include the relevant information there.

Reply:
I just missed it before pushing, will delete it 👍